source/views/sis/balances.tsx

```diff
@@ -13,13 +13,14 @@ import {BalancesShapeType, useBalances} from '../../lib/financials'
 import * as c from '@frogpond/colors'
 import {sto} from '../../lib/colors'
 import {useNavigation} from '@react-navigation/native'
-import {NoCredentialsError, useCredentials} from '../../lib/login'
+import {NoCredentialsError, useUsername} from '../../lib/login'
 
 const DISCLAIMER = 'This data may be outdated or otherwise inaccurate.'
 
 export const BalancesView = (): JSX.Element => {
 	let navigation = useNavigation()
-	let credentials = useCredentials()
+	let {data: usernameData} = useUsername()
```

Comment:
```suggestion
	let {data: username = ''} = useUsername()
```

I think we can do this now?